cmake/CPM.cmake

```diff
@@ -210,11 +210,25 @@ if(NOT CPM_INDENT)
   )
 endif()
 
+if(NOT CPM_LEFT_LOG)
```

Comment:
+1 for the suggestion to use `cpm_message()` and allow specialization on that in a single location. 

However, on keeping this approach the choise of variables naming is odd imho. I would prefer `CPM_LOG_PREFIX` and `CPM_LOG_POSTFIX` to be more self explanatory on the intent.